components/date_input/README.md

```diff
@@ -0,0 +1,161 @@
+---
+description: Date input allows users to enter an already known date
+title: Date Input
+status: Draft
+---
+
+# Date Input
+
+Date input fields allow users to enter an already known date
+
+### Required
+
+## Last Updated:
+
+## Visual
+Date Input Visual:
+![Date Input](https://github.com/bcgov/design-system/blob/master/components/date_input/images/date_input.png?raw=true)
+
+## Requirements
+* This component must be used according the the standard outined in this document. Any feedback should be directed to the feedback forum.
+
+## How to Use This
+*	Use when users need to enter an easily memorable date
+
+## Don't Use This For
+*	Dates that cannot be easily remembered
+*	When users have to schedule something in the future, a calendar dropdown may be better.
+
+## Rationale
+Using three separate text fields is the fastest way for users to enter dates. Dropdown options cause accessibility difficulties.
+
+## Behaviour
+
+* Do not auto advance to next input field. This causes difficulties for users relying on keyboard controls.
+* When in focus input field should outline with blue.
+* Fields should have error controls such as not accepting more values than required, rejecting false values, and rejecting outlier values.
+
+## Accessibility
+This checkbox has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
+
+### Screenreaders
+* Proper ALT labels are included
+* Language tag has been set to english
+
+### Colour Contrast
+* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text
+
+### Keyboard Navigation
+* Supports tab navigation for use without a mouse
+
+### Learn More
+* Contacts
+* Online Tools
+* Online Resources
+* Gov Resources
+
+## Prototype with This Component
+Download and use the B.C. government Design System library to help build prototypes and wireframes for your website or application.
+
+The library is constantly updated and currently available in the following tools:
+
+*	[Sketch](https://sketch.cloud/s/Q0bkG)
+* [UxPin and others]
+
+## Code
+
+### HTML
+```
```

Comment:
You can often get syntax highlighting by adding the code type right after the third back-tick. For example add `html` or `css`. 
```html
<h1> Hello </h1>
```

Reply:
Awesome, thanks for the tip!